R/visr.R

```diff
@@ -168,7 +168,7 @@ visr.survfit <- function(
 # Y-label ----------------------------------------------------------------------
 
   if (is.null(y_label) & is.character(fun)){
-    y_label <- base::switch(
+    y_label <- switch(
```

Comment:
keep base::switch as discussed. We never know if switch will occur in other packages.